File: src/test/fixtures/python/opaque_boolean.py
# flagged — opaque boolean
def flaggedPositionalBoolean():
    configure(True)


# flagged — opaque boolean
def flaggedPositionalBooleanAmongOthers():
    process(1, False)


# clean — not flagged by opaque boolean
def suppressedKeywordArgument():
    configure(retries=True)


# clean — not flagged by opaque boolean
def suppressedNonCallUsage():
    return True

File: src/test/fixtures/cpp/opaque_boolean.cpp
// flagged — opaque boolean
void flaggedPositionalBoolean() {
    configure(true);
}

// flagged — opaque boolean
void flaggedPositionalBooleanAmongOthers() {
    process(1, false);
}

// clean — not flagged by opaque boolean
void suppressedLabeledAggregateField() {
    configure(Settings{.retries = true});
}

// clean — not flagged by opaque boolean
bool suppressedNonCallUsage() {
    bool ok = true;
    return ok;
}
